README.md

```diff
@@ -190,6 +190,28 @@ Simply run `make install` - this will compile the provider and install it to `~/
 Requirements:
 - [Terraform](https://www.terraform.io/downloads.html) 0.13
 - [Go](https://golang.org/doc/install) 1.15+ (to build the provider plugin)
+- [gnu-sed](https://formulae.brew.sh/formula/gnu-sed)
+
+### gnu-sed
+
+This provider uses [GNU sed](https://www.gnu.org/software/sed/) as part of the build toolchain, in both Linux and macOS. This provides consistency across OSes. If you are building this on macOS, you will need to install [gnu-sed using brew](https://formulae.brew.sh/formula/gnu-sed).
```

Comment:
Honestly, why not just build with a docker image and call it done? We can dictate the terms of a build

Reply:
If the user wants to build and run in macOS (like we do currently), this is still an issue.

Reply:
I guess I need to do the same for Xray? :-/

Reply:
@danielmkn You should for the time being. We will pick a Docker image as the 'official' image to use for building the provider. Let's get this and the Xray provider released first before we do that.